internal/dev_server/model/mocks/observer.go

```diff
@@ -3,11 +3,11 @@
 //
 // Generated by this command:
 //
-//	mockgen -destination mock_observer/observer.go -package mock_observer . Observer
+//	mockgen -destination mocks/observer.go -package mocks . Observer
 //
 
-// Package mock_observer is a generated GoMock package.
-package mock_observer
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
Paired with @mike-zorn on batching all the mock objects in models together in one directory and re-ran codegen.